test/e2e_node/runtime_conformance_test.go

```diff
@@ -76,7 +76,7 @@ var _ = SIGDescribe("Container Runtime Conformance Test", func() {
 					})
 				})
 
-				f.It(testCase.description+"", f.WithNodeConformance(), f.WithFlaky() /* https://github.com/kubernetes/kubernetes/issues/134998 */, func(ctx context.Context) {
+				f.It(testCase.description+"", f.WithNodeConformance(), f.WithDisruptive(), f.WithSerial(), func(ctx context.Context) {
```

Comment:
@SergeyKanzhelev can a disruptive and serial test be considered in node conformance?

I typically see node conformance for parallel/fast tests but I'm not sure if we have any other tests that contain all three of these tags..

Anyway this is non blocking.

Reply:
"Disruptive" in generally meaningless. The better name for it is "Flaky". I do not think we will ever run all disruiptive tests isolated from each other

Reply:
Thanks! I do see some tests in our serial lanes with disruptive, serial and node conformance so this is actually done.

Reply:
hm, this is interesting definition of disruptive. Many serial jobs wouldn't expect a lot of activity in the cluster happening at the same time. So kind of disruptive since node should be given completely to that test execution.

Reply:
Disruptive can also be a test that intentionally induces load (there's one that triggers GC IIRC), most cases of disruptive are also serial, but not all serial are disruptive.

E.G. the tests that setup the test registry must be serial currently, but it's not disruptive.


cc @pohly @aojea

Reply:
just to reiterate the point, if test can fail if anything else is happening at the same time (Serial) - it likely a prod disruption for the entire node. I do not know how Disruption tag is being used, but I was always relaxed on advising on it and never asked to add it explicitly.